### PR TITLE
python/raml package

### PR DESCRIPTION
**Pull request by kalefranz**

### CODE REVIEW

**Comment by ilanschnell** [comment]
Is it possible to fun the tests in a cleaner way.  Something like:
```
# run_test.py
import raml
assert raml.test().wasSuccessful()
```

**Comment by kalefranz** [comment]
Something like that @ilanschnell ?

**Comment by ilanschnell** [comment]
Yeah

File: python/raml/bld.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set INCLUDE=%LIBRARY_INC%
+set LIBPATH=%LIBRARY_LIB%
+set LIB=%LIBRARY_LIB%
+
+python setup.py install
+if errorlevel 1 exit 1
+
+copy %LIBRARY_BIN%\yaml.dll %SP_DIR%\raml\ext\
```
Comment (kalefranz) [inline python/raml/bld.bat]: @ilanschnell note the `%SP_DIR%\raml\ext\` directory.
compare to: https://bitbucket.org/kalefranz/yaml/src/default/setup.py?fileviewer=file-view-default#setup.py-20

File: python/raml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  name: raml
+  version: 0.11.7
+
+build:
+  number: 0
+  # Windows has issues updating stuff if conda is using files itself. Copy rather than link.
+  no_link: .*\.(pyd|dll)        [win]
+
+source:
+  hg_url: ssh://hg@bitbucket.org/kalefranz/yaml
```
Comment (groutr) [inline python/raml/meta.yaml]: I think sources should be fetched via http/https instead of ssh.

**Pull request merged**